README.md

```diff
@@ -26,14 +26,20 @@ Have a clean repository that meets quality coding standards (file structure, nam
 Have a quality README (with unique name, description, technologies used, screenshot, and link to deployed application).
 
 ## Web Application Preview
-![Main screen](<Screenshot 2023-10-08 231308.png>)
-![Random movie chosen](<Screenshot 2023-10-08 231622.png>)
-![Youtube trailer](<Screenshot 2023-10-08 231637.png>)
+![Main screen](<webpage-preview-1>)
+![Random movie chosen](<webpage-preview-2>)
+![Youtube trailer](<webpage-preview-3>)
 
-## Link to Deployed Applications
+## Link to Deployed Application
 To use this webpage and start your own movie quest, please use this URL: https://ajayshans.github.io/moviequest/
 
+## Acknowledgments
+
+Youtube API: https://developers.google.com/youtube/v3/docs 
```

Comment:
I think these need the asterisks as well

Reply:
![Screenshot 2023-10-09 at 5 48 13 pm](https://github.com/ajayshans/moviequest/assets/69303149/40a54292-38ca-46c6-bb39-b2b116228720)